RoaringBitmap/src/main/java/org/roaringbitmap/ParallelAggregation.java

```diff
@@ -180,10 +187,15 @@ public static RoaringBitmap or(RoaringBitmap... bitmaps) {
    * @return the symmetric difference of the bitmaps
    */
   public static RoaringBitmap xor(RoaringBitmap... bitmaps) {
+    IMemoryManager memoryManager =
+        bitmaps.length == 0 ? DefaultMemoryManager.INSTANCE : bitmaps[0].memoryManager;
```

Comment:
if empty, returns an empty bitmap (-> no memorymanager should be involved).
if not empty, the Collector should choose dynamically the proper memorymanager. As the first bitmap may be a dumb/empty RoaringBitmap with a DefaultMemoryManager.INSTANCE, while other non-empty bitmaps could have a custom memoryManager.